macros/NextFlex_fullKr.init.mac

```diff
@@ -22,7 +22,7 @@
 ### ACTIONS
 /nexus/RegisterRunAction      DefaultRunAction
 /nexus/RegisterEventAction    DefaultEventAction
-/nexus/RegisterTrackingAction DefaultTrackingAction
+/nexus/RegisterTrackingAction OpticalTrackingAction
```

Comment:
Why are you changing this example to save all the optical photons? Take into account that this macro is run every time we run the automatic tests, and saving the optical photons increases the execution time.

Reply:
This was to check if the optical fiber barrel works as intended (For this of course the optical photons are of interest). Considering the automatic tests, this can be changed back.

Reply:
Fabian, we think that the tests are failing because of the large RAM consumption due to this change. Please, change it back to `DefaultTrackingAction` and push the commit, to see if that's the case.